### DISCUSSION TITLE
post/spring-batch-performance/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#39] [Announcements]
# post/spring-batch-performance/

벌크 I/O 처리를 통한 성능 향상 기법을 다룹니다.

https://tech.kakaopay.com/post/spring-batch-performance/

**Comment by pay-dan-dy** [comment]
배치 성능 향상에 않은 도움이 될 것 같아요! 감사합니다!

**Comment by sinminseok** [comment]
좋은 글 감사합니다. 도움 많이 됐어요!
